perllib/FixMyStreet/App/Form/Waste.pm

```diff
@@ -55,17 +56,25 @@ sub validate {
     my $email = $self->field('email');
     my $phone = $self->field('phone');
     return 1 unless $email && $phone;
+    my $text_updates = $self->field('extra_bulky_text_updates');
 
     my $c = $self->c;
     my $cobrand = $c->cobrand->moniker;
     my $is_staff_user = ($c->user_exists && ($c->user->from_body || $c->user->is_superuser));
     my $staff_provide_email = (ref $self) =~ /Garden/ && ($c->cobrand->moniker eq 'kingston' || $c->cobrand->moniker eq 'sutton');
 
     $email->add_error('Please provide an email address')
-        unless $email->is_inactive || $email->value || ($is_staff_user && !$staff_provide_email);
-
+        unless $email->is_inactive || $email->value || ($is_staff_user && !$staff_provide_email)
+        || (FixMyStreet::SMS->parse_username($phone->value)->{may_be_mobile} && $cobrand eq 'peterborough');
```

Comment:
This line seems to make the email address optional on the "About you" page if I provide a mobile number, but I can't complete a booking without an email address: 
<img width="548" alt="image" src="https://github.com/mysociety/fixmystreet/assets/4776/38db8122-041a-4afd-811c-7e2ba2d56df9">

Reply:
Fix to this in ba1e0155f9f0afe8ec29ac485f1102d9dad002d3 May need more discussion...